prov/net/src/xnet_ep.c

```diff
@@ -666,6 +666,9 @@ int xnet_endpoint(struct fid_domain *domain, struct fi_info *info,
 	if (ret)
 		goto err1;
 
+	if (xnet_io_uring)
+		xnet_prefetch_rbuf_size = 0;
```

Comment:
This function (ep creation) doesn't look like the correct place to make this change to a global variable